crates/cargo-util-schemas/src/restricted_names.rs

```diff
@@ -238,6 +238,37 @@ pub(crate) fn validate_feature_name(name: &str) -> Result<()> {
     Ok(())
 }
 
+pub(crate) fn validate_path_base_name(name: &str) -> Result<()> {
+    let what = "path base name";
+    let mut chars = name.chars();
+
+    let Some(first_char) = chars.next() else {
+        return Err(ErrorKind::Empty(what).into());
+    };
+
+    if !first_char.is_alphabetic() {
```

Comment:
Is there a reason we went with ASCII only names in the RFC?  For myself, I'd prefer we try to aim for consistency between our different validation rules which would likely mean delegating to `validate_name`

Reply:
I was following the wording in the RFC. I switched to `validate_name` to be consistent with the rest of Cargo.